tools/generate_llvm_coverage.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
```

Comment:
Please add copyright header as in `generate_tarpaulin_coverage.sh`

Reply:
will do